test/r401_test.js

```diff
@@ -375,6 +375,54 @@ describe('#R4 v4.0.1', () => {
   });
 });
 
+describe('PatientSource meta.profile checking', () => {
```

Comment:
I know it is tedious, but I would recommend adding this test suite for all the models.  The reason I tend to do that is because the modelinfo is actually _used_ differently in each one (i.e., it differs not only in the FHIR content, but in how the modelinfo schema itself is used; there is no consistency between models). I've had a few times when I implemented something here and it worked great in R4 but went down in flames in STU3 or DSTU2 -- all because the modelinfo was done just a little differently.